src/TodoListItem.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function TodoListItem({ todo, handleDeleteTodo }) {
```

Comment:
한번 더 구조분해할당을 해줄 수 있겠네요 :)
```suggestion
function TodoListItem({ todo: { id, content }, handleDeleteTodo }) {
```

Reply:
이부분에서 질문이 있습니다.

`TodoList`에서 `todo`를 넘겨줄때 지금처럼 `todo` 객체를 통으로 `TodoListItem` 컴포넌트에 넘기는 방법과,
아래처럼 `TodoListItem`에서 필요한 항목들만 전달하는 방법중 어느 방법이 더 효율적인? 좋은? 방법인가요?
```js
<TodoListItem
          key={todo.id}
          id={todo.id}
          content={todo.content}
          handleDeleteTodo={handleDeleteTodo}
        />
```

Reply:
지금과 같이 객체 내에 프로퍼티가 몇 없을 경우에는 어떤 방법으로 해도 상관은 없을 것 같아요~ 

대신 객체 내에 프로퍼티가 많아지면 어떨까요? 이 경우엔 하나씩 맵핑해주는 것보다 객체를 넘겨서 그 내부에서 처리하는 것도 방법일 것 같네요